src/hooks/useWithdrawLiquidityHandler.tsx

```diff
@@ -88,132 +124,183 @@ export const useWithdrawLiquidityHandler = () => {
       return
     }
 
-    if (depPool === null) {
-      return
-    }
+    if (process.env.NEXT_PUBLIC_ENVIRONMENT === 'local') {
+      if (posManager === null) {
+        return
+      }
 
-    try {
-      let tx = await depPool.removeLiquidity(amt, ZEROMIN, ZEROMIN, accountInfo.address, {
-        gasLimit: 10000000,
-      })
-      return await tx.wait()
-    } catch (e) {
-      return e
+      try {
+        const WithdrawReservesParams = {
+          cfmm: process.env.NEXT_PUBLIC_CFMM_ADDRESS,
+          protocol: 1,
+          amount: amt,
+          amountsMin: [100, 200, 300],
+          to: accountInfo.address,
+          deadline: ethers.constants.MaxUint256,
+        }
+        let tx = await posManager.withdrawReserves(WithdrawReservesParams)
+        return await tx.wait()
+      } catch (e) {
+        return e
+      }
+    } else {
+      if (depPool === null) {
+        return
+      }
+
+      try {
+        let tx = await depPool.removeLiquidity(amt, ZEROMIN, ZEROMIN, accountInfo.address, {
+          gasLimit: 10000000,
```

Comment:
should make this an env or global variable

Reply:
Added it as an environment variable in env files